src/commands/install-docker-compose.yml

```diff
@@ -71,7 +71,7 @@ steps:
           --silent --show-error --location --fail --retry 3 \
           "$DOCKER_COMPOSE_SHASUM_URL"
 
-        FILENAME=$(cat docker-compose-$PLATFORM-x86_64.sha256 | awk '{ print $NF }')
+        FILENAME=$(cat docker-compose-$PLATFORM-x86_64.sha256 | awk '{ print $NF }') | sed 's/^\*//'
```

Comment:
Tests are failing due to `FILENAME` resolving to an empty value. This occurs because `sed` is sitting outside the sub-shell invocation.

Reply:
Oops, bad copy-paste. Glad tests caught it!